README.md

```diff
@@ -99,27 +99,34 @@ name the -B and -T options allow a mapping file to be specified to
 rename branches and tags (respectively). The syntax of the mapping
 file is the same as for the author mapping.
 
-Content filtering
+Plugins
 -----------------
 
-hg-fast-export supports filtering the content of exported files.
-The filter is supplied to the --filter-contents option. hg-fast-export
-runs the filter for each exported file, pipes its content to the filter's
-standard input, and uses the filter's standard output in place
-of the file's original content. The prototypical use of this feature
-is to convert line endings in text files from CRLF to git's preferred LF:
+hg-fast-export supports plugins to manipulate the file data and commit
+metadata. The plugins are enabled with the --plugin option. The value
+of said option is a plugin name (by folder in the plugins directory),
+and optionally, and equals-sign followed by an initialization string. 
+
+There is a readme accompanying each of the bundled plugins, with a 
+description of their usage. To create a new plugin, one must simply
+add a new folder under the `plugins` directory, with the name of their
+new plugin. Inside, there must be an `__init__.py` file, which contains
+at a minimum:
 
```

Comment:
For distributions which package fast-export (at least Debian, Gentoo and Homebrew does) and install into system directories we should perhaps add a command line flag to add an external directory to the search path. That way you can use the installed fast-export with a custom plugin.